### PR TITLE
Kovan deprecation, replace RPCs, add code notices

### PR DESCRIPTION
**Pull request by dwightjl**
- Migrate remaining Kovan RPC endpoints to Rinkeby
- Change remaining Kovan code samples to Rinkeby
- Add code audit notices

### CODE REVIEW

**Comment by dwightjl** [comment]
Rebased to main

File: docs/Using Price Feeds/feed-registry.html [hunk 1]
```diff
@@ -34,9 +34,6 @@
 ```solidity Mainnet
 latestRoundData(0x514910771AF9Ca656af840dff83E8264EcF986CA, 0x0000000000000000000000000000000000000348)
 ```
-```solidity Kovan
```
Comment (aelmanaa) [inline docs/Using Price Feeds/feed-registry.html:37]: should we remove the config from the page while kovan is still running?

File: docs/Using Price Feeds/get-the-latest-price.html [hunk 2]
```diff
@@ -75,7 +75,7 @@
 
 ## Python
 
-This example uses [Web3.py](https://web3py.readthedocs.io/en/stable/) to retrieve feed data from the [ETH / USD feed](https://rinkeby.etherscan.io/address/0x8A753747A1Fa494EC906cE90E9f37563A8AF630e) on the Rinkeby testnet.
+This example uses [Web3.py](https://web3py.readthedocs.io/en/stable/) to retrieve feed data from the [BTC / USD feed](https://rinkeby.etherscan.io/address/0x8A753747A1Fa494EC906cE90E9f37563A8AF630e) on the Rinkeby testnet.
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply docs/Using Price Feeds/get-the-latest-price.html]: Fixed.

File: docs/Using Price Feeds/feed-registry.html (hunk 1 above)
Comment (dwightjl) [reply docs/Using Price Feeds/feed-registry.html]: Restored

**Review by aelmanaa — APPROVED** [review]

**Pull request merged**